platform/mikro-e/Makefile.mikro-e

```diff
@@ -42,7 +46,7 @@ endif
 
 MODULES += core/net core/net/mac core/net/llsec
 
-CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES)
+CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES) slip.c
```

Comment:
you can append the slip.c filename to CONTIKI_TARGET_SOURCEFILES variable as it is defined in the same makefile